src/utils/components/PolicyForm/DeleteInterfaceModal.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { useNMStateTranslation } from 'src/utils/hooks/useNMStateTranslation';
+
+import {
+  ActionList,
+  ActionListItem,
+  Button,
+  ButtonVariant,
+  Modal,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
+import { NodeNetworkConfigurationInterface } from '@types';
+
+import { capitalizeFirstLetter } from './utils';
+
+type DeleteInterfaceModalProps = {
+  closeModal: () => void;
+  isOpen?: boolean;
+  policyInterface: NodeNetworkConfigurationInterface;
+  onSubmit: (policyInterface: NodeNetworkConfigurationInterface) => void;
+};
+
+const DeleteInterfaceModal: React.FC<DeleteInterfaceModalProps> = ({
+  closeModal,
+  isOpen,
+  policyInterface,
+  onSubmit,
+}) => {
+  const { t } = useNMStateTranslation();
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    onSubmit(policyInterface);
+    closeModal();
+  };
+
+  return (
+    <Modal
+      className="ocs-modal"
+      onClose={closeModal}
+      variant="medium"
+      position="top"
+      title={t('Delete NodeNetworkConfigurationPolicyInterface')}
```

Comment:
Please, add "?" to the end of the title string. I was fixing this in many modals in our other repo ;) (same for the [other title](https://github.com/nmstate/nmstate-console-plugin/pull/19/files#diff-06769e0d19e91d6d6217e0d2623118aa48486b82b2ddd6db4c66c4d1d4432ccbR64) ;)

Reply:
Good catch